src/shared/Internal.Shared.Tasks/buildMultiTargeting/Internal.Shared.Tasks.props

```diff
@@ -0,0 +1,22 @@
+﻿<Project>
```

Comment:
These are all `<UsingTask/>` elements. If those are different between `build` and `buildMultiTargeting`, something is very wrong. Restore the old file structure: Have both .props files `<import/>` on that's shared. Fine to improve on the unusual `Tasks.tasks` name e.g. `Tasks.props`.